src/models/SessionReturnRecord.ts

```diff
@@ -0,0 +1,67 @@
+import { ReturnSQSEvent } from "./ReturnSQSEvent";
+import { Constants } from "../utils/Constants";
+
+export interface NamePart {
+	type: string;
+	value: string;
+}
+
+export interface PersonName {
+	nameParts: NamePart[];
+}
+export class SessionReturnRecord {
+	constructor(data: ReturnSQSEvent, expiresOn: number) {
+		this.userId = data.user.user_id;
+		switch (data.event_name) {
+			case Constants.AUTH_IPV_AUTHORISATION_REQUESTED:{
+				this.clientName = data.client_id!;
+				this.redirectUri = data.component_id!;
+				this.userId = data.user.user_id;
+				this.userEmail = data.user.email;
+				this.ipvStartedOn = data.timestamp;
+				this.expiresOn = expiresOn;
+				break;
+			}
+			case Constants.F2F_YOTI_START:{
+				this.userId = data.user.sub!;
+				this.journeyWentAsyncOn = data.timestamp;
+				break;
+			}
+			case Constants.IPV_F2F_CRI_VC_CONSUMED:{
+				this.readyToResumeOn = data.timestamp;
+				break;
+			}
+			case Constants.AUTH_DELETE_ACCOUNT:{
+				this.accountDeletedOn = data.timestamp;
+				this.clientName = "";
+				this.redirectUri = "";
+				this.userEmail = "";
+				this.nameParts = [];
+				break;
+			}
+			default:{
+				break;
+			}
+		}
+	}
+
+    userId: string;
+
+    userEmail?: string;
+
+    nameParts?: PersonName[];
+
+    clientName?: string;
+
+    redirectUri?: string;
+
+    ipvStartedOn?: number;
+
+    journeyWentAsyncOn?: number;
+
+    readyToResumeOn?: number;
+
+    accountDeletedOn?: number;
+
+    expiresOn?: number;
```

Comment:
could we set this one as `expiryDate` pls as [SessionEvent table already got TTL for that field ](https://github.com/alphagov/di-ipvreturn-api/blob/a7195864d162244c8a88b7bcf9d39493d8b61552/infra-l2-dynamo/template.yaml#L74)

Reply:
Done.